framework/src/seclink/seclink.c

```diff
@@ -124,7 +139,7 @@ int sl_init(sl_ctx *hnd)
 	handle->fd = fd;
 
 	struct seclink_req req = {.req_type.comm = NULL, 0};
-	SL_CALL(handle, SECLINKIOC_INIT, req);
+	SL_CALL2(handle, SECLINKIOC_INIT, req, SL_FREE_HANDLE(handle));
```

Comment:
SL_CALL2 is not good to understand.
There is "SL_CALL_NORET" on below, like this, Would you change like 'SL_CALL_RET'?